common/app/model/SupportedUrl.scala

```diff
@@ -13,7 +13,11 @@ object SupportedUrl {
 
   def fromFaciaContent(fc: PressedContent): String =
     fc match {
-      case curatedContent: CuratedContent => s"/${curatedContent.properties.href.getOrElse(fc.card.id)}"
+      case curatedContent: CuratedContent =>
+        s"/${curatedContent.properties.webUrl
+          .map(webUrl => webUrl.replaceFirst("^[a-zA-Z]+://www.theguardian.com/", ""))
```

Comment:
`webUrl` is in the form of `https://www.theguardian.com/<path>`, so we need to remove `https://www.theguardian.com/` to avoid 404s.

![Screenshot 2022-06-13 at 21 10 57](https://user-images.githubusercontent.com/19683595/173436860-430db71b-bb50-40f2-971e-a1017315b43e.png)